app/services/cloud.js

```diff
@@ -94,7 +94,7 @@ CloudService.prototype.signinUser = function signinUser(credentials, done) {
       protocol: this.authenticatorAddress.protocol,
       hostname: this.authenticatorAddress.hostname,
       port: this.authenticatorAddress.port,
-      pathname: path.join(this.authenticatorAddress.path, 'auth')
+      pathname: path.join(this.authenticatorAddress.path, 'tokens')
```

Comment:
I would rename from `authenticatorAddress` to `apiGatewayAddress` since the target service has changed. I think you can do similar for all `authenticator` occurrence.

Reply:
Done!